[MMM-15567] introduce an environment that supports public custom metric (#875)
datarobot/datarobot-user-models: public_custom_metrics_templates_environments/python39_llm_templates/README.md

# Python 3.9 Custom Metrics Templates Drop-In Environment

This environment is used as the default environment in custom metrics templates.

> Hosted custom metrics created from templates have this environment attached.

Reviewer: ```suggestion
Hosted custom metrics created from templates will run in this environment.
```
